tests/Import.spec.ts

```diff
@@ -27,7 +26,7 @@ test.describe(`Testfälle für den Benutzerimport": Umgebung: ${process.env.ENV}
 
   test.beforeEach(async ({ page }) => {
     await test.step('Einloggen und zu Benutzerimport navigieren', async () => {
-      await page.goto(FRONTEND_URL);
+      await page.goto('/');
```

Comment:
Bitte das als Veriable irgendwo einmal setzten z.B.: WEBSEITE_ROOT = '/' damit das überall im Code wieder verwendbar ist :)

Reply:
In der playwright.config.ts ist die base url definiert; baseURL: FRONTEND_URL,
Mit der Zeile 29 wird genau dahin gesprungen. Die goto Methode hat einen Parameter vom typ string der Pflicht ist. Man könnte anstatt '/' auch '' setzen, dieses dann aber als extra Variable zu machen? Bin mir da nicht sicher.

Reply:
Beim nächsten refactoring muss die login-Methode überarbeitet werden, so dass solche Sachen wie das goto aus allen Tests rausfällt.

Reply:
Momentan wird das nur in 6 Files verwendet, es kann sein dass es zukünftig an anderen Stellen oder bein neuen Tests benötigt wird. Ist das einfach die LandingPage also gotto('/') leitet auf die LandingPage um?

Wenn das der Fall, würde ich persönlich das einfach konfigurieren LANDING_PAGE = '/'